docs/en/operations/settings/settings.md

```diff
@@ -365,7 +365,22 @@ throws an exception.
 
 ## input_format_null_as_default {#settings-input-format-null-as-default}
 
-Enables or disables using default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats).
+For text input formats initializes `NULL` fields with default values if data type of these fields is not nullable.
+This setting is applied to `INSERT ... VALUES` queries.
+
+## insert_null_as_default {#insert_null_as_default}
+
+Enables the insertion of [default values](../../sql-reference/statements/create/table.md#create-default-values) instead of [NULL](../../sql-reference/syntax.md#null-literal) into clumns with not [nullable](../../sql-reference/data-types/nullable.md#data_type-nullable) data type. 
```

Comment:
```suggestion
Enables the insertion of [default values](../../sql-reference/statements/create/table.md#create-default-values) instead of [NULL](../../sql-reference/syntax.md#null-literal) into columns with not [nullable](../../sql-reference/data-types/nullable.md#data_type-nullable) data type. 
```